lib/profileAgents.js

```diff
@@ -301,6 +292,44 @@ exports.delegateCapabilities = async (
   return Promise.all(promises);
 };
 
+exports.delegateCapabilityInvocationKey = async ({profileAgent, invoker}) => {
+  const {
+    controller: {seed, keystore: keystoreId},
+    capabilityInvocationKey
+  } = profileAgent;
+  console.log('ProfileAgent', JSON.stringify(profileAgent, null, 2));
+
+  const controller =
+    await CapabilityAgent.fromSecret({handle: 'primary', secret: seed});
+
+  const keystore = await kms.getKeystore({id: keystoreId});
+  const keystoreAgent = kms.getKeystoreAgent(
+    {capabilityAgent: controller, keystore});
+  const key = await keystoreAgent.getAsymmetricKey(
+    {id: capabilityInvocationKey, type: 'Ed25519VerificationKey2018'});
+  await utils.setKeyId({key});
+
+  const request = {
+    referenceId: `profileAgentCapabilityInvocationKey`,
```

Comment:
Don't need an ES6 String Template here. This `referenceId` differs from the pattern that we had and that might just be okay.

Reply:
I'd be inclined to say that single quotes and backticks are equivalent in this case.  If you feel strongly about this, I'll make an issue on our linting rules to make this an error.

Reply:
An issue to our linting rules would be fine.

Reply:
+1 to linting rule fix and requiring single quotes when not using any variables. I would imagine there's overhead to using backticks when not needed ... and in general, we should just be consistent and say not to use them when they aren't needed.